### PR TITLE
Adding changes for getting value from map

### PR DESCRIPTION
**Pull request by kanjoe24**

### CODE REVIEW

File: tests/src/ut_test_control_plane.c [hunk 1]
```diff
@@ -321,6 +328,18 @@ void test_ut_control_performExit( void )
...
+void test_ut_control_get_map_value( void )
+{
+    UT_LOG("test_ut_control_get_map_value()\n");
+    UT_ASSERT(UT_Control_GetMapValue(numericMaptable, "three", 0) == 3);
+}
+
+void test_ut_control_get_map_string( void )
+{
+    UT_LOG("test_ut_control_get_map_string()\n");
+    UT_ASSERT_EQUAL(UT_Control_GetMapString(numericMaptable, 1), "one");
```
Comment (Ulrond) [inline tests/src/ut_test_control_plane.c:340]: `UT_ASSERT_EQUAL_STRING()` am surprised this worked, I don't think it should.

**Review by Ulrond — CHANGES_REQUESTED** [review]
need to compare strings.

File: tests/src/ut_test_control_plane.c (hunk 1 above)
Comment (Ulrond) [reply tests/src/ut_test_control_plane.c:340]: are you not getting compiler warnings? Is all the warnings enabled in the makefile?

Comment (kanjoe24) [reply tests/src/ut_test_control_plane.c]: done

**Review by Ulrond — APPROVED** [review]
looks good

**Pull request merged**